Doc/reference/simple_stmts.rst

```diff
@@ -801,10 +801,10 @@ The :keyword:`from` form uses a slightly more complex process:
 Examples::
 
    import foo                 # foo imported and bound locally
-   import foo.bar.baz         # foo.bar.baz imported, foo bound locally
-   import foo.bar.baz as fbb  # foo.bar.baz imported and bound as fbb
-   from foo.bar import baz    # foo.bar.baz imported and bound as baz
-   from foo import attr       # foo imported and foo.attr bound as attr
+   import foo.bar.baz         # foo, foo.bar, and foo.bar.baz imported, foo bound locally
+   import foo.bar.baz as fbb  # foo, foo.bar, and foo.bar.baz imported, foo.bar.baz bound as fbb
+   from foo.bar import baz    # foo, foo.bar, and foo.bar.baz imported, foo.bar.baz bound as baz
+   from foo import attr       # foo and foo.attr imported and foo.attr bound as attr
```

Comment:
```suggestion
   from foo import attr       # foo imported and foo.attr bound as attr
```
I think the intent here is that `foo.attr` is a regular attribute of `foo`, not a submodule.

Reply:
Would foo.attr still be technically imported?

Reply:
No, see line 793 above. If the attr already exists, it doesn't try to import it.

Reply:
got it! refactored